en/docs/developer-portal/manage-application.md

```diff
@@ -51,3 +51,62 @@ Let's get started!
 8. Click **:// CURL** to copy the cURL command template for generating the access token. You can add the consumer key and consumer secret to the URL and invoke the token endpoint to generate an API access token.
 
     ![cURL command](../assets/img/developer-portal/manage-applications/curl-command.png){.cInlineImage-half}
+
+
+## Revoke access tokens
+
+In the event of a theft, security violation, or a precaution, an admin can revoke an access token. Choreo provides you with a revoke token endpoint. You can use a utility like a cURL to invoke this endpoint and revoke your access token.  
```

Comment:
Would it be more meaningful if we combined the first two sentences to give out the meaning that Choreo provides the revoke token endpoint to allow users to revoke an access token?
Suggestion:
In the case of theft, security violation, or precaution, Choreo allows an admin to revoke an access token via the revoke token endpoint.

Reply:
done